Orange/widgets/visualize/owmosaic.py

```diff
@@ -281,8 +281,11 @@ class OWMosaicDisplay(OWWidget):
     variable2 = ContextSetting("", exclude_metas=False)
     variable3 = ContextSetting("", exclude_metas=False)
     variable4 = ContextSetting("", exclude_metas=False)
+    variable_color = ContextSetting("", exclude_metas=False)
     selection = ContextSetting(set())
 
+    color_var = None
```

Comment:
It is not needed, I thought that I needed it, but then I have forgotten to erased it. Any yes, Lint reports that this variable is unused.